backend/api/user.go

```diff
@@ -28,6 +28,15 @@ func user(c echo.Context) error {
 	return c.JSON(http.StatusOK, "User get handler")
 }
 
+func logout(c echo.Context) error {
+	// Setting MaxAge<0 means delete cookie now.
+	u := http.Cookie{
+		Name: "UserAuth"}
+	u.Expires = time.Unix(0, 0)
+	c.SetCookie(&u)
+	return c.JSON(http.StatusOK, "Old Cookie Deleted. Logged Out!\n")
+	// !!!!!!!!!!!Refersh Token STILL needs to be added to the database
+}
```

Comment:
The two cookies that need to be invalidated upon logout are "accessToken" and "refreshToken". You ought to use `auth.ACCESS_TOKEN_COOKIE_KEY` and `auth.REFRESH_TOKEN_COOKIE_KEY` as the names (they are defined contants)